lib/bin/lwm2m_carrier/include/lwm2m_carrier.h

```diff
@@ -490,36 +1123,44 @@ int lwm2m_carrier_request(int request);
 void lwm2m_carrier_time_read(int32_t *utc_time, int *utc_offset, const char **tz);
 
 /**
- * @brief Function to read current UTC time
+ * Read of the OMA LwM2M `Device` object `Current Time` resource (/3/0/13).
```

Comment:
I think this would be better here and below
```suggestion
 * Read the OMA LwM2M `Device` object `Current Time` resource (/3/0/13).
```